components/dropdown/dropdown.tsx

```diff
@@ -169,11 +174,17 @@ const Dropdown: DropdownInterface = props => {
     alignPoint = true;
   }
 
+  const getDropdownPlacements = () =>
+    getPlacements({
+      arrowPointAtCenter: typeof arrow === 'object' && arrow.pointAtCenter,
+    });
```

Comment:
```suggestion
```suggestion

  const builtinPlacements = getPlacements({
    arrowPointAtCenter: typeof arrow === 'object' && arrow.pointAtCenter,
  });
```

然后下面直接 `builtinPlacements={builtinPlacements}`，这样是不是更直观一点。
```